3.26.0 ERROR post install npx husky install

## Description 🐜

It seems the last build isn't working. I've spent an half hour trying to undersand why Vercel was suddenly unable to deploy my project... until I saw an update of next-auth had been released.

## How to reproduce ☕️

Run npm i next-auth 

## Screenshots / Logs 📽

```
❯ npm i next-auth

> next-auth@3.26.0 postinstall /Users/rosaygaspard/Documents/dev/Web/ReactJS/nextjs/ac-eup/node_modules/next-auth
> npx husky install

npx: installed 1 in 1.241s
.git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
npm WARN ac-eup@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! next-auth@3.26.0 postinstall: `npx husky install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the next-auth@3.26.0 postinstall script.
```
## Environment 🖥

System:
    OS: macOS 11.3.1
    CPU: (4) x64 Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz
    Memory: 128.24 MB / 8.00 GB
    Shell: 5.8 - /usr/local/bin/zsh
  Binaries:
    Node: 14.11.0 - /usr/local/bin/node
    npm: 6.14.8 - /usr/local/bin/npm
  Browsers:
    Chrome: 91.0.4472.101
    Firefox: 88.0.1
    Safari: 14.1

Also happened while deploying to Vercel

Seeing the same thing locally:

```
$ npm install --save next-auth

> npx husky install

npx: installed 1 in 5.989s
.git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
...
...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! next-auth@3.26.0 postinstall: `npx husky install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the next-auth@3.26.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
...
```

Tried the previous version and it worked fine:

```
+ next-auth@3.25.0
added 80 packages from 111 contributors and audited 811 packages in 16.43s
found 0 vulnerabilities
```

--- a/package.json
+++ b/package.json
@@ -29,7 +29,6 @@
     "./errors": "./dist/lib/errors.js"
   },
   "scripts": {
-    "postinstall": "npx husky install",
     "build": "npm run build:js && npm run build:css",
     "build:js": "node ./config/build.js && babel --config-file ./config/babel.config.js src --out-dir dist",
     "build:css": "postcss --config config/postcss.config.js src/**/*.css --base src --dir dist && node config/wrap-css.js",

